lib/tree.rb

```diff
@@ -1,38 +1,61 @@
 class NoApplesError < StandardError; end
 
+
 class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+  attr_accessor :height, :age, :apples, :alive
 
   def initialize
+    @height = 0
+    @age = 0
+    @apples = []
+    @alive = true
+    @colors = ['red', 'green', 'yellow']
```

Comment:
Should apple colors be a property of the tree, or of the apple?  I guess you could make a case either way.  But given what you're doing with it here, it might make sense to move the random color- and size-generation into `Apple#initialize`